src/WorkflowManager/MonaiBackgroundService/appsettings.Development.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
```

Comment:
Are these needed now that it is going to be a background service?

Reply:
removed all the appsettings.json and launchsettings.json